### PR TITLE
Add Elastio terraform deployment automation example

### PR DESCRIPTION
**Pull request by Veetaha**
[Rendered README](https://github.com/elastio/contrib/tree/feat/elastio-terraform-deployment/elastio-terraform-deployment)

Closes https://github.com/elastio/elastio/issues/9409

### CODE REVIEW

File: elastio-terraform-deployment/module/main.tf [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+resource "aws_cloudformation_stack" "elastio_nat_provision_stack" {
+  count = var.elastio_nat_provision_stack == null ? 0 : 1
+
+  name = "elastio-nat-provision-lambda"
+  template_url = join(
+    "/",
+    [
+      "https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com",
```
Comment (anelson) [inline elastio-terraform-deployment/module/main.tf:41]: There shouldn't be any issue w/ deployments to some other region using the `us-east-2` bucket, right?

File: elastio-terraform-deployment/module/variables.tf [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+variable "elastio_tenant" {
+  description = "Name of your Elastio tenant. For example `mycompany.app.elastio.com`"
+  type        = string
+  nullable    = false
+}
+
+variable "elastio_cloud_connectors" {
+  description = "List Cloud Connectors to deploy"
```
Comment (anelson) [inline elastio-terraform-deployment/module/variables.tf:15]:
```suggestion
  description = "List of regions where Cloud Connectors are to be deployed, and VPC and subnet(s) to use"
```

**Review by anelson — APPROVED** [review]
LGTM!

File: elastio-terraform-deployment/module/main.tf (hunk 1 above)
Comment (Veetaha) [reply elastio-terraform-deployment/module/main.tf:41]: Yes, the Cloudformation template URL can be in any region

**Pull request merged**